documentation/modules/rn-2.2.adoc

```diff
@@ -40,6 +52,15 @@ As a temporary workaround, the `Importer` pod is removed at the end of the preco
 
 This issue only affects the `Importer` pod log and warm migration. Cold migration and the `virt-v2v` logs are not affected.
 
+.Unclear error message for failed migration of VMs with no operation system
+
+When migration plans for VMs with no operation system fail, the error message does not indicate the reason for the failure. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[*BZ#2008846*])
+
+.In a migration plan created from the CLI using names instead of IDs for network and storage mapping and for VMs, the mapping is shown as unavailable when the plan is opened in the {project-full} web console.
+
+If you create a migration plan from the CLI and use names for storage and network mapping and for VMs instead of their IDs, the plan shows the mapping as unavailable when you open it in the {project-full} web console. In addition, the edit and duplicate features do not work. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
```

Comment:
Use the proper names for the CRs. 

`If a `Plan` CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-full} web console. You cannot edit or clone the migration plan. (link)`